### ISSUE TITLE
Screen reader does not read out who wrote the post

### ISSUE THREAD

**Issue by Lizzy-j** [fac19/week5-DGHP#43]
The screenreader only picks up where aria labels have been added. Have you thought about using an aria-label and assigning the user's name with string interpolation so the screen reader picks up on this?
